pom.xml

```diff
@@ -14,6 +14,11 @@
   <name>Jenkins SSH Slaves plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
 
+  <properties>
+    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
+    <findbugs-maven-plugin.failOnError>true</findbugs-maven-plugin.failOnError>
```

Comment:
The [user property](http://gleclaire.github.io/findbugs-maven-plugin/check-mojo.html#failOnError) is `findbugs.failOnError` do not hijack the expected property and move it to a different name